Source/Core/Core/AchievementManager.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#ifdef USE_RETRO_ACHIEVEMENTS
+
+#include "Core/AchievementManager.h"
+#include "Common/HttpRequest.h"
+#include "Common/Thread.h"
+#include "Config/AchievementSettings.h"
+#include "Core/Core.h"
+
+AchievementManager::AchievementManager()
+    : m_runtime(), m_is_runtime_initialized(false), m_login_data{.response{.succeeded = 0}},
```

Comment:
We generally prefer initializing in the class body if possible. The following example gives all the initializers needed to get rid of this list of initializations

```cpp
class AchievementManager
{
public:
  // ...
private:
  // ...
  rc_runtime_t m_runtime = {};
  bool m_is_runtime_initialized = false;
  rc_api_login_response_t m_login_data = {};
  std::string m_password;
  std::mutex m_lock;
  std::thread m_achievements_thread;
  Common::Event m_post_requested;
}
```